pyccel/codegen/printing/ccode.py

```diff
@@ -944,6 +946,22 @@ def _print_Import(self, expr):
             source = source.name[-1]
         else:
             source = self._print(source)
+        if expr.target:
+           dtype = expr.target.pop().name
+           if source.startswith('stc'):
```

Comment:
I think you should keep the `/` so that you don't have problems if a user imports from a file that they created called something like `stc_pythonfile`